core/src/templates/components/main-nav/main-nav.json

```diff
@@ -8,9 +8,7 @@
     {
       "href": "#",
       "text": "First level item",
-      "attributes": {
-        "class": "su-secondary-nav__item--expanded"
-      },
+      "current": true,
```

Comment:
Oh, I meant to revert this back to the original json. The new schema here is for testing.